### PR TITLE
Add terradoc

### PR DESCRIPTION
**Pull request by kassianh**
Recreated README with terradoc, checked documentation for errors or inconsistencies.

### CODE REVIEW

**Comment by kassianh** [comment]
@mariux Is it ok to remove markdown-link-check? Otherwise checks fail due to dead links that aren't actually dead..

**Comment by soerenmartius** [comment]
> @mariux Is it ok to remove markdown-link-check? Otherwise checks fail due to dead links that aren't actually dead..

yip that's fine - we're working on a new version that doesn't break the CI but just highlights broken links instead

File: README.tfdoc.hcl [hunk 1]
```diff
@@ -0,0 +1,1033 @@
+header {
+  image = "https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg"
+  url   = "https://mineiros.io/?ref=terraform-module-template"
```
Comment (soerenmartius) [inline README.tfdoc.hcl:3]: still references the template, this should be changed!

**Review by soerenmartius — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -438,141 +450,149 @@ See [variables.tf] and [examples/] for details and use-cases.
...
+  Example:
 
   ```hcl
   sql_databases = [{
     name = "db"
   }]
   ```
 
-  Each `sql_databases` object accepts the following fields:
+  The object accepts the following attributes:
```
Comment (mariux) [inline README.md:551]: @thiesen this should be fixed in terradoc i assume... ? when it's a list it should state "each object"..

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**